### PR TITLE
Add WMS layer extended from Layer

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

**Comment by edellucien** [comment]
Removed from static fcts

File: src/layers/WMSLayer.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    // Add click callback
+    if (options.onClick) {
+      this.onClick(options.onClick);
+    }
+  }
+
+  /**
+   * Get features infos' Url.
+   * @param {ol.layer} layer ol.layer (https://openlayers.org/en/latest/apidoc/module-ol_layer_Layer.html)
```
Comment (mhaertwig) [inline src/layers/WMSLayer.js]: Please remove

File: src/layers/WMSLayer.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        .getGetFeatureInfoUrl(coord, resolution, projection, {
+          info_format: 'application/json',
+        });
+    }
+    return false;
+  }
+
+  /**
+   * Get features infos for WMS layer.
+   * @param {ol.layer} layer ol.layer (https://openlayers.org/en/latest/apidoc/module-ol_layer_Layer.html)
```
Comment (mhaertwig) [inline src/layers/WMSLayer.js]: Can be removed.

File: src/layers/WMSLayer.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+   * @param {ol.coordinate} coord ol.coordinate (https://openlayers.org/en/latest/apidoc/module-ol_coordinate.html)
```
Comment (mhaertwig) [inline src/layers/WMSLayer.js]: ol.Coordinate

**Pull request merged**